src/persist/src/operators/input.rs

```diff
@@ -136,11 +137,13 @@ mod tests {
             p.into_inner()
         });
 
-        let mut p = MemPersister::from_inner(dataz);
+        // TODO: I don't get what this second scope is meant to illustrate?
+        let mut p = FilePersister::from_inner(dataz);
         let recv = timely::execute_directly(move |worker| {
             let ((mut handle, cap), recv) = worker.dataflow(|scope| {
                 let persister = p.create_or_load(Id(1)).unwrap();
                 let (input, stream) = scope.new_persistent_unordered_input(persister);
+                // TODO what actually happens in this line?
```

Comment:
It connects the stream to the channel so we can extract the contents after the dataflow is finished and assert on them